serializations.md

```diff
@@ -312,18 +335,17 @@ purging them from the tree.
 
 #### Name service commitment
 ```
-[ <hash>    :: binary()
```

Comment:
If name removed from body returned from user API, [intended usage may need update here](https://github.com/aeternity/protocol/blob/6614f009a3d033e3f8aec6a381d16def766f7eaa/epoch/api/naming_system_api_usage.md#update).

Reply:
It is just the serialization that is removed. The API objects look the same.